### ISSUE TITLE
Fetching information over internet should be done with https

### ISSUE THREAD

**Issue by jvoisin** [squentin/gmusicbrowser#107]
It would be great if information about artists/albums/songs/… and covers would be downloaded over http**s** instead of plain http.

**Comment by anterguy** [comment]
Fully agree! https should be used whenever possible if it's not toomuch of a hassle ;-)

**Comment by RomanHargrave** [comment]
if gmb is using any relatively abstract library for making HTTP requests, this should be perfectly feasible

**Comment by squentin** [comment]
Yes, it should be easy. It uses one of 3 small libraries (simple_http*pm). The simplest one,  simple_http.pm won't support it, but it doesn't work well anyway, and is only used as a last recourse.

I just have to check each website used to see if they support https, and figure out what I do with simple_http.pm.
